acquisitions/src/main/resources/thunderjet/mod-organizations/reusable/create-acq-unit.feature

```diff
@@ -0,0 +1,25 @@
+@ignore
+Feature: Create ACQ Unit
+  # parameters: id?, name, code, status?
```

Comment:
It seems like status is not being used here?

Reply:
Thanks, fixed the arguments